File: lam.hs
-- Normal function in Haskell
add :: Int -> Int -> Int
add x y = x + y

-- Lambda function in Haskell
addLambda :: Int -> Int -> Int
addLambda = \x y -> x + y

-- Defining results varibales
result1 :: Int
result1 = add 3 2

result2 :: Int
result2 = addLambda 3 2 

main :: IO ()
main = do
  print result1 -- normal function
  print result2

File: lam.js
//Regular normal function

function add(x,y){
	return x + y;
}

// Arrow function (lambda function)
const AddFun = (x,y) => x + y;

// Defining results
let result1 = add(2,4);
let result2 = AddFun(2,3); 

// Printing on the console
console.log("result from normal function: " + result1);
console.log("result from Arrow/Lambda function: " + result2)


